tests/utils/test_databricks_utils.py

```diff
@@ -200,3 +202,16 @@ def test_databricks_params_throws_errors(ProfileConfigProvider):
     ProfileConfigProvider.return_value = mock_provider
     with pytest.raises(Exception):
         databricks_utils.get_databricks_host_creds()
+
+
+def test_is_in_databricks_runtime():
+    with mock.patch("sys.modules", new={**sys.modules, "pyspark": mock.MagicMock()}):
+        with mock.patch("sys.modules", new={**sys.modules, "pyspark.databricks": mock.MagicMock()}):
```

Comment:
Is this really correct?

Reply:
Have you tried this?

```suggestion
    with mock.patch("sys.modules", new={**sys.modules, "pyspark": mock.MagicMock(), "pyspark.databricks": mock.MagicMock()}):
```